### ISSUE TITLE
Planned custom fieldtypes

### ISSUE THREAD

**Issue by mmikkel** [mmikkel/DashCols-Craft#9]
Here's a list of custom/plugin FieldTypes I'm currently looking at supporting in DashCols:
- [Introvert](https://github.com/iainurquhart/craft_introvert)
- [SetTable](https://github.com/engram-design/SetTable)
- [Template Select](https://github.com/sjelfull/Craft-TemplateSelect)
- [Randomm](https://github.com/sjelfull/Craft-Randomm)
- [Icons](https://github.com/fruitstudios/Icons)
- [Territories](https://github.com/johndwells/craft.territories)
- [Range Slider](https://github.com/jack-pallot/craft-range-slider-fieldtype/)
- [Listing Section](http://straightupcraft.com/craft-plugins/listing-section)
- [ButtonBox](https://github.com/supercool/Button-Box) (suggested by @engram-design)  

Know another fieldtype you'd like to see added? Please comment.

**Comment by engram-design** [comment]
How about [ButtonBox](https://github.com/supercool/Button-Box)? Particularly the stars field would sweet

**Comment by mmikkel** [comment]
@engram-design Yup, that one looks both plausible and worthwhile. Added it to the list. Thanks!
